src/snowcli/app/dev/docs/templates/usage.rst.jinja2

```diff
@@ -18,7 +18,7 @@ Arguments
 
 {%- if arguments %}
 {% for param in arguments %}
-:samp:`{% for p in param.opts %}{{ p }}{{ ", " if not loop.last }}{% endfor %} {{ '{' }}{{ param.name }}{{ '}' }}`
+:samp:`{% for p in param.opts %}{{ p }}{{ ", " if not loop.last }}{% endfor %} {{ '{' }}{{ param.make_metavar() }}{{ '}' }}`
```

Comment:
Can you show an example of rendered output?

Reply:
example is present in PR description